CMake/GenPkgConfig.cmake

```diff
@@ -91,8 +91,15 @@ function(gen_pkgconfig_vars)
   endif()
 
   # Set the output variables
-  string(REPLACE ";" " " cflags "${cflags}")
+  list(JOIN cflags " " cflags)
```

Comment:
`list(JOIN` is not supported on all versions of CMake that we support building with, so this will need to stay as replace for now.

Reply:
Didn't notice that Folly supports such ancient CMake versions. Fixed.